students/FredB/session01/breakme.py

```diff
@@ -0,0 +1,19 @@
+#Assignment1
+#FredBallyns
+
+def NameError():
+#print(undefinedText)
+
+def TypeError():
+a=5
+b="5"
+#c=a+b
+
+def SyntaxError():
+#"
+
+def AttributeError():
+List = 5
```

Comment:
did this compile and run?

you need to indent the code inside a function:

```
def attribute_error():
    List = 5
    LIst.append(4)
```
then you need to call the function to get it to run:

```
attribute_error()
```

And you don't want to call your function "AttributeError", as that will over wright the built in name for the Exception.